changelog.txt

```diff
@@ -1,5 +1,22 @@
 *** Changelog ***
 
+## 4.16.0 - 2023-07-13
+
+### Added
+- Ability to set conditions on admin notices based on a date range
+- Add disclamer with the reason that Sensei Pro can't be updated when license is not active
+- Add quiz generation using AI button in quiz block
+- Enable to select how to generate lessons when a course is created [#6999]
```

Comment:
Here we should link to the PRs. You can see examples from the last release.

Reply:
The ones without links should not be on the changelog. 
Removed here fcb535725c5ac7b0decddf55435891d81901ad3d